### DISCUSSION TITLE
Help is required

### DISCUSSION THREAD

**Discussion by elmans1961** [Chia-Network/chia-blockchain#4854] [Support]
My log files look like
2021-05-13T23:26:17.354 timelord chia.timelord.timelord   : WARNING  Will not infuse 035a2c8f359f2a8b0d8ed675e8752fe3d4207c9fe6e164d458539a87d1b579d0 because its reward chain challenge is not in the chain
2021-05-13T23:26:17.357 timelord chia.timelord.timelord   : WARNING  Will not infuse 4d447509a0b275ce04ca9d2e4d51b2fb8a0432737a3edca7e560b7f5158f827a because its reward chain challenge is not in the chain
2021-05-13T23:26:17.360 timelord chia.timelord.timelord   : WARNING  Will not infuse 1dc8cebd1a3d0374b44d9f2b0ff613416a59fb1733c7acce4f1e93624916e42c because its reward chain challenge is not in the chain
Something is wrong with me though I upgraded to v 1.5.5 and migrated all old files to a new version with a command line.My full node in GUI shows wallet,farmer,timeLord and full node information but no info of the harvester,Is it  right?Please,help me!

**Comment by zhengzhang2040** [comment]
I also meet this , and I have been kept farming for 2 weeks ,100T+ , haven't seen any coins so far.

**Reply by elmans1961** [reply]
with more than 100T on which you can place +1090  plots you should mine more than a coin every ten days according to estimation.

**Reply by zhengzhang2040** [reply]
but I've been farming for half a month , haven't seen any coins .

**Reply by cyrus104** [reply]
I'm in the same boat but I only have 700plots.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
